crates/ffi/src/erc6492_client.rs

```diff
@@ -0,0 +1,46 @@
+use crate::ffi::Erc6492Error;
+use alloy::{
+    network::Ethereum,
+    primitives::{Address, Bytes, B256},
+    providers::ReqwestProvider,
+};
+
+pub struct Erc6492Client {
+    provider: ReqwestProvider<Ethereum>,
+}
+
+impl Erc6492Client {
```

Comment:
I understand that I can use this client to verify any type of accounts, EOAs and SA, correct?

Reply:
Yeah, you can verify all signature types with this